ipwb/indexer.py

```diff
@@ -306,19 +304,18 @@ def getCDXJLinesFromFile(warcPath, **encCompOpts):
 
             objJSON = json.dumps(obj)
 
-            cdxjLine = '{0} {1} {2}'.format(originaluri_surted,
-                                            timestamp, objJSON)
+            cdxjLine = f'{originaluri_surted} {timestamp} {objJSON}'
             cdxjLines.append(cdxjLine)  # + '\n'
         return cdxjLines
 
 
 def generateCDXJMetadata(cdxjLines=None):
     metadata = ['!context ["http://tools.ietf.org/html/rfc7089"]']
     metaVals = {
-        'generator': "InterPlanetary Wayback v.{0}".format(ipwbVersion),
-        'created_at': '{0}'.format(datetime.datetime.now().isoformat())
+        'generator': f'InterPlanetary Wayback v.{ipwbVersion}',
```

Comment:
Should we have a dot after `v`?

```suggestion
        'generator': f'InterPlanetary Wayback v{ipwbVersion}',
```

Reply:
Yes, I think so, but am not tied to it. Should we even have a "v" and not just state "InterPlanetary Wayback 0.2020.06.24" in the indexes?

This also makes me wonder if we need some sort of regression/adaptation to indexes created with older versions, e.g., the metadata first character change.

Reply:
Either way is fine, whether or not we include `v`, but we need to be consistent about it.

We do need to revisit old index files and adjust every now and then as things evolve.

Reply:
`ipwb --version` returns the version number appended without a "v". I believe the above code is the only instance where we generate new data with an ipwb version number. I would prefer removing the "v." altogether.

Reply:
I was wrong. We still use it in GitHub releases.